### ISSUE TITLE
need to do it manually?

### ISSUE THREAD

**Issue by sanjaylama** [Yurich84/laravel-vue-spa-skeleton#15]
Very promising work.  But so after generating module if we have to add more fields we have to manually add each of the fields in to model, migration and write the codes in controller and views files as well. That is lot of work and it won't be CRUD anymore. It will be more a starter boilerplate then.
Also, does it handle table join at all or that question is not applicable?
It sounds more a laravel vue module file structure generator rather than CRUD.
Also, in the vue folders there maybe some unnecessary files which can be managed in better ways. Maybe.

Thanks anyways.

**Comment by Yurich84** [comment]
@sanjaylama I deliberately did not make it possible to create additional fields. Because they in the same application can have different properties and behaviors in different places. This is a different validation, depending on other conditions. It's easier to set them up manually.
If these are text fields, then it is not difficult to copy them. If these are selects, sliders, etc., then you still have to redo it.
What I have done allows you to quickly start developing a separate module avoiding the same steps. And create modules in the same style.
In my opinion, the ability to generate all the necessary fields with a command will not lead to much acceleration of development.

**Comment by sanjaylama** [comment]
Yes definitely it helps and a good package considering you have a module approach but it does have a lot of promising future if you can integrate that so was just suggesting.
Even if you can mimic what is done by this, then definitely would be one of the great github resources..
https://github.com/InfyOmLabs/laravel-generator

same thing maybe just with vue...
Thank anyways. Great work.
